scripts/setup-htpass-idp.sh

```diff
@@ -10,14 +10,11 @@ if [[ ! -f "htpasswd" ]]; then
   touch htpasswd
 fi
 
-htpasswd -b htpasswd customer-admin ${PASSWORD}
-echo user added customer-admin ${PASSWORD}
-
 htpasswd -b htpasswd test-user ${PASSWORD}
 echo user added test-user ${PASSWORD}
 
 oc delete secret htpasswd-secret -n openshift-config
 oc create secret generic htpasswd-secret --from-file=htpasswd=htpasswd -n openshift-config
 oc patch oauth cluster --type=merge -p '{ "spec": { "identityProviders": [{ "name": "htpasswd_provider", "challenge": true, "login": true, "mappingMethod": "claim", "type": "HTPasswd", "htpasswd": { "fileData": { "name": "htpasswd-secret" } } }] } }'
 
-oc adm groups add-users dedicated-admins customer-admin
+oc adm groups add-users dedicated-admins test-user
```

Comment:
I don't think we want to make any changes to this script.
Intention was to remove only the hard-coded customer-admin user.
We still need "customer-admin" user in this script to test permissions of a user in "dedicated-admins" group.
And we need "test-user" to test permissions of a user that wasn't added to any group explicitly.

JIRA description might have been a little misleading, sorry for that.

Reply:
I have reverted it back